runtime/runtime.go

```diff
@@ -203,17 +205,17 @@ func reportMetric(
 
 // interpreterRuntime is an interpreter-based version of the Flow runtime.
 type interpreterRuntime struct {
-	defaultConfig Config
+	DefaultConfig Config
```

Comment:
Does the debugger need to write this field? If not, maybe expose this via a getter? Might also help to prevent future accidental changes probably.
Fine with either way though.

Reply:
Looking at https://github.com/onflow/flow-emulator/pull/164 again: It seems like it's not actually needed? I might have directly modified the config at some point, and then later introduced `SetDebugger` instead 🤔